src/anniversary/sf33rd/Source/Game/IOConv.c

```diff
@@ -0,0 +1,106 @@
+#include "unknown.h"
+#include "common.h"
+#include "sf33rd/AcrSDK/common/mlPAD.h"
+
+extern u32 ioconv_table[24][2];
+
+void keyConvert(void) {
+    typedef struct {
+        u8 state;
+        u8 anstate;
+        u16 kind;
+        u32 sw;
+        u32 sw_old;
+        u32 sw_new;
+        u32 sw_off;
+        u32 sw_chg;
+        u32 sw_repeat;
+        PAD_STICK stick[2];
+    } IOPad;
+    IOPad* pad;    
+    u32 currSw;
+    s32 i;
+    s32 j;
+    s32 repeat_on;
+
+    repeat_on = 0;
+    if ((Debug_w[0x2B]) && (mpp_w.inGame) && (Game_pause == 0)) {

```

Comment:
Redundant braces here and throughout the file

```suggestion
    if (Debug_w[0x2B] && mpp_w.inGame && (Game_pause == 0)) {
```

Reply:
Also the preferred way to format `if` statements in the project is to separate them from outer code with a single blank line. Haven't configured it as a linter/formatter rule yet, should probably do that